metricbeat/module/vsphere/datastore_cluster/_meta/data.json

```diff
@@ -0,0 +1,27 @@
+{
+    "@timestamp": "2016-05-23T08:05:34.853Z",
+    "service": {
+        "address": "https://localhost:8980/sdk",
+        "type": "vsphere"
+    },
+    "event": {
+        "dataset": "vsphere.generic",
```

Comment:
Please refer to this [issue](https://github.com/elastic/obs-infraobs-team/issues/1391#issuecomment-2116846101), I don't think `event.dataset: vsphere.generic` will work on integration side due to the metricset name is different from the value of `event.dataset`! 

We already faced this issue in nagios_xi integration. 

The solution could be set value `event.dataset: vsphere.datastore_cluster` and introduce one more field called `event.provide: vsphere.generic` to avoid pipeline or system tests failure in integration?

Reply:
Lets please ensure we get the actual event for data.json for these
We will not see these issues then.